tests/tests/test_train_schedule.py

```diff
@@ -235,32 +235,32 @@ def test_etcs_schedule_stop_brakes_result_never_reach_mrsp(
     #   dodges this limit and starts under "high" MRSP (288 km/h), and the guidance curve change at 220 km/h is also
     #   noticeable.
     # In practice, check noticeable points of the braking curves (with the stops already checked)
-    offset_first_high_speed = 14_509_017
-    offset_first_brake_220_kph_speed = 17_544_856
+    offset_first_high_speed = 14_509_004
```

Comment:
nit: for clarity and to match other namings
```suggestion
    offset_start_first_brake_high_speed = 14_509_004
```

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/11616/commits/d328ef56112a6bcbb3a997d5966f1b8f63722afa